e2e-test/cypress/integration/currencies_spec.js

```diff
@@ -54,7 +55,7 @@ describe("Overview Page", function () {
     cy.get("[data-test=cancel]").click();
   });
 
-  it("Sets the currency of a new project to EUR and checks if the Euro sign is displayed", function () {
+  it("Sets the currency of a new project to EUR and checks if the Euro sign is displayed", function() {
     cy.createProject("Test", "Test", standardBudget);
```

Comment:
use one project for all currency tests. Put the create project in a before() and save the project id. Use the other tests to check how its done quickly.